packages/twenty-front/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx

```diff
@@ -106,11 +107,7 @@ export const ScrollWrapper = ({
           id: contextProviderName,
         }}
       >
-        <StyledScrollWrapper
-          ref={scrollableRef}
-          className={className}
```

Comment:
why are we removing the scrollHide?

Reply:
We use scrollHide specifically to hide NavigationDrawer's scroll, but we have new requirements :). We are to show the scroll but with clear paddings